testcafe/tests/pages/drive/drive-model.js

```diff
@@ -98,8 +100,14 @@ export default class DrivePage {
     this.cozySelectionbarBtnShare = this.cozySelectionbar
       .find('button')
       .withText('SHARE') //!FIX ME : do not use text! Do not use .nth(x) because the buttons count changes if one or several files/folders are selected
-    this.modalDelete = Selector('[class*="c-modal"]').find('div')
-    this.modalDeleteBtnDelete = this.modalDelete.find('button').nth(2) //REMOVE
+    this.cozySelectionbarBtnRename = this.cozySelectionbar
+      .find('button')
+      .withText('RENAME') //!FIX ME : do not use text! Do not use .nth(x) because the buttons count changes if one or several files/folders are selected
```

Comment:
What's the hold-up for fixing this?

Reply:
https://trello.com/c/22GedOwK/1850-%F0%9D%90%92-check-de-tous-les-selecteurs-sortir-des-classes-et-faire-un-fichier-communs-avec-tous-les-selecteurs-en-exports : I will have a better idea of which cozyBar buttons shows up, in order to use relevant (and easy to read) selector